.gitignore

```diff
@@ -72,3 +72,6 @@ google-services.json
 
 # MSC
 .DS_Store
+
+# ignore .import folder
+*.import
```

Comment:
This also ignores files, which ironically enough is not what we want :)

either specify the path or use a glob pattern for directories: `**/.import/`